addons/binding/org.openhab.binding.onkyo/README.md

```diff
@@ -15,15 +15,30 @@ This binding supports only one thing: The Onkyo AV Receiver.  All supported Onky
 
 This binding can discover the supported Onkyo AV Receivers. At the moment only the following models are supported:
 
+* TX-NR414
+* TX-NR509
+* TX-NR515
+* TX-NR525
 * TX-NR535
+* TX-NR555
+* TX-NR535
+* TX-NR616
 * TX-NR626
+* TX-NR636
 * TX-NR646
+* TX-NR656
+* TX-NR717
+* TX-NR727
+* TX-NR747
+* TX-NR818
+* TX-NR828
+* TX-NR838
 
 ## Binding Configuration
 
 The binding can auto-discover the Onkyo AVRs present on your local network. The auto-discovery is enabled by default. To disable it, you can create a file in the services directory called onkyo.cfg with the following content:
 
-`org.openhab.onkyo:enableAutoDiscovery=false`
+```org.openhab.onkyo:enableAutoDiscovery=false```
```

Comment:
There seems to be Markdown Eclipse plugin available

https://github.com/winterstein/Eclipse-Markdown-Editor-Plugin/

Maybe this could be added to openHAB IDE environment (Eclipse Installer) to be installed automatically.